src/reducers/dataReducer.js

```diff
@@ -59,11 +59,13 @@ export default function dataReducer(state = {}, action) {
           state,
         );
       }
+      // Otherwise merge with existing data
+      const mergedData = assign(previousData, data);
```

Comment:
This will prevent removing a param from an object that is removed elsewhere - what is the goal here?

Reply:
The PR is meant to revert back an existing functionality that has been later removed (while kept in rrf - firebase), data merging.
When for a given collection you change the query to fetch new data (or next page for that matters) new data should be merged with existing data, at the moment is getting replaced. 
What the PR is doing is just reverting back the files to their previous status (as mentioned in the issue open https://github.com/prescottprue/redux-firestore/issues/294)